submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,272 @@
+
+{% include image.html
+
+src= 'src'
+ alt='Processing solid fuel into rocket fuel'
+ caption='Processing solid fuel into rocket fuel'
+
+%}
+
+We see that a single unit rocket fuel contains 100MJ of energy while the required ingredients of 10 solid fuel already holds 120MJ, and the 10 light oil can be made into an 11th solid fuel to provide almost 12MJ more. Hence, we see that processing solid fuel into rocket fuel makes you lose energy, before even accounting for processing energy. However, as we will cover later on, rocket fuel has advantages over solid fuel that may outweigh the energy losses. Furthermore, it is a necessary ingredient if we want to make nuclear fuel, which may increase the overall energy gains despite the losses at this stage.
+
+## Going nuclear
+
+Nuclear fuel is still pretty experimental in the real world, but Factorio offers a standardized variant that is made from regular rocket fuel and uranium-235. Like with our examination of rocket fuel, the recipe itself is enough to determine energy gains:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing rocket fuel into nuclear fuel'
+ caption='Processing rocket fuel into nuclear fuel'
+
+%}
+
+We see that nuclear fuel offers an impressive energy output of 1.21GJ, which is [a reference](https://wiki.factorio.com/Nuclear_fuel), but also around 10 times as much energy as the solid fuel first used to make the rocket fuel. This means that if you were to switch from burning coal directly to making it into nuclear fuel to burn instead, your coal demand would drop to a tiny fraction, to as little as 1/20 as before. 
+
+## Making the most of your uranium
+
+Despite its massive energy gains and the resulting coal and oil savings, nuclear fuel might not be perfect. The uranium-235 that it uses could instead be used to make uranium fuel cells, which provide a different energy situation:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing uranium-235 into uranium fuel cells'
+ caption='Processing uranium-235 into uranium fuel cells'
+
+%}
+
+We see that while uranium fuel cells can only be used inside nuclear reactors, they produce energy on the order of GJ instead of MJ, and the energy yield is not simply boosted but multiplied via the reactor neighbor bonus. Hence, up to hundreds of times more energy is gained from the same piece of uranium-235 if used in a nuclear reactor instead of nuclear fuel. This would suggest that if one has a uranium-235 supply already, they should just switch to nuclear power and electric machines instead of burner devices. However, it is also worth noting that nuclear fuel is unlocked alongside Kovarex Enrichment, which allows for easy production of uranium-235. This means that using uranium efficiently stops being an issue around the time you unlock nuclear fuel.
```

Comment:
> Hence, up to hundreds of times more energy is gained

Maybe show a quick calculation for this using 0, 2 and 3 neighbors (1 neighbor isn't seen often and 4 neighbors of course prevents automatic fuel insertion and extraction).